### DISCUSSION TITLE
Text to Speech rate limits

### DISCUSSION THREAD

**Discussion by kholiavko-roman** [deepgram/community#711] [General help] [answered]
Hello, I am curious if it's possible to increase the rate limit on Aura text-to-speech model on a different service plan?

Because by default it's only [2 parallel requests](https://developers.deepgram.com/reference/api-rate-limits) and we are hitting the rate limit even in the dev mode. In production, it will be a disaster for us. 

Thanks.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by michellelychan** [comment, marked as answer]
Thanks @kholiavko-roman ! I'm Michelle and I'm the product manager working on Deepgram's text-to-speech product. I'd love to have a chat with you to learn more about your needs. Just sent you an email on your GH profile for next steps.

**Reply by kholiavko-roman** [reply]
Thanks.
